integration_tests/e2e/updateFunctionalityFull.cy.ts

```diff
@@ -35,7 +35,7 @@ context('Update functionality - Full flow', () => {
 
   describe('Happy path scenarios', () => {
     beforeEach(() => {
-      cy.task('getCiagPlan', 'A3260DZ')
+      cy.task('stubGetInductionLongQuestionSet', 'A3260DZ')
```

Comment:
For the cypress integration tests I've changed the stub from a CIAG API based stub to our new Induction API stubs